Database/Updates/Shard/2017-08-20-00-vw-ace-wielded-object.sql

```diff
@@ -1,3 +1,4 @@
+use ace_shard;
```

Comment:
We historically and intentionally do not have using statements.  I understand it makes things easier to mess up, but that's the policy.

Reply:
I see the value in `use` statements, is this something worth revisiting to make easier folks pulling down the repo and using the typical values?